course-mw/course-actors-common/src/main/java/org/sunbird/learner/util/BatchLocalCache.java

```diff
@@ -0,0 +1,16 @@
+package org.sunbird.learner.util;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class BatchLocalCache {
```

Comment:
this class is not used anywhere -- can this be removed ??

Reply:
@ShankarTarento Class is not required please go ahead and delete